models/models.js

```diff
@@ -18,10 +17,40 @@ exports.fetchReviewById = (review_id) => {
         if (!review.rows[0]) {
             return Promise.reject({
                 status: 404,
-                msg: `Review ${review_id} could not be found`,
+                msg: 'Not Found',
             }
             );
         }
         return review.rows;
     });
-};
\ No newline at end of file
+};
+
+utilCheckReviewExist = (review_id) => {
+    return db.query(`SELECT * FROM reviews WHERE review_id = $1;`, [review_id]).then(res => {
+        if (res.rows.length === 0) {
+            return Promise.reject({
+                status: 404,
+                msg: 'Review not Found',
+            }
+            );
+        }
+    });
+};
+
+exports.fetchCommentsByReviewId = (review_id) => {
+    return utilCheckReviewExist(review_id)
+        .then(() => {
+            return db.query('SELECT * FROM comments WHERE review_id = $1 ORDER BY created_at ASC;', [review_id]);
+        })
+        .then(comments => {
+            if (!comments.rows[0]) {
+                return Promise.reject({
+                    status: 404,
+                    msg: 'Review has no comment',
+                }
+                );
+            }
```

Comment:
Like I described in my previous comment, this isn't a 404, just an empty result.